docs/_docs/plugins.md

```diff
@@ -68,6 +68,24 @@ You have 3 options for installing plugins:
   </p>
 </div>
 
+### The jekyll_plugins group
+
+Jekyll gives this particular group of gems in your `Gemfile` a different
+treatment. Any gem included in this group is loaded before Jekyll starts
+processing the rest of your source directory.
+
+A gem included here will be activated even if its not explicitly listed under
+the `plugins:` key in your site's config file.
+
+<div class="note warning">
+  <p>
+    Gems included in the <code>:jekyll-plugins</code> group are activated
+    regardless of the <code>--safe</code> mode setting. Be careful of what
+    gems are included under this group!
```

Comment:
I'm a little weary of the FUD (fear, uncertainty, and doubt) that "Be careful!" leads to. Could we rephrase this to be less fear-mongery? "Be aware" perhaps?

Reply:
oops didn't see you comment, but totally agree with you. Will fix this right away

Reply:
fixed in https://github.com/jekyll/jekyll/commit/52c34060e3311c1d2a369f02b4d2319c1b283f7b

Reply:
I didn't even notice this. 🙈  Thanks for pointing it out @parkr.